src/Nancy/DefaultStaticContentConfigurationProvider.cs

```diff
@@ -0,0 +1,30 @@
+﻿namespace Nancy
+{
+    using Nancy.Configuration;
+
+    /// <summary>
+    /// Provides the default configuration for <see cref="StaticContentConfiguration"/>. 
+    /// </summary>
+    public class DefaultStaticContentConfigurationProvider : NancyDefaultConfigurationProvider<StaticContentConfiguration>
+    {
+        private readonly IRootPathProvider rootPathProvider;
+
+        /// <summary>
+        /// Creates and instance of DefaultStaticContentConfigurationProvider

```

Comment:
`Creates and instance of <see cref="DefaultStaticContentConfigurationProvider" />`